connection.js

```diff
@@ -23,6 +22,7 @@ var trans       = require('./transaction');
 var plugins     = require('./plugins');
 var rfc1869     = require('./rfc1869');
 var outbound    = require('./outbound');
+var ResultStore = require('haraka-results');
```

Comment:
Just curious, why move this down? The general format you'll find in most files is a section of node.js builtin modules, then a section of NPM modules, and finally a section of local (./) modules.

Reply:
It was a mistake, I based this off the older one and reverted a lot of stuff, this should've been one of them, fixing now